extensions/default/src/Panels/Toolbox/ToolboxContainer.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useReducer } from 'react';
+import { useToolbar } from '@ohif/core';
+import { toolboxReducer } from './toolboxReducer';
+import { ToolboxUI } from './ToolboxUI';
+
+const initialState = {
+  activeTool: null, // Tracks the current active tool
+  toolOptions: {}, // Stores the options for each tool
+};
+
+// Todo: make this configurable
+const itemsPerRow = 4;
+
+function ToolboxContainer({ servicesManager, commandsManager, buttonSectionId, title }) {
+  const { stateSyncService } = servicesManager.services;
+  const { onInteraction, toolbarButtons } = useToolbar({
+    servicesManager,
+    buttonSection: buttonSectionId,
+  });
+
+  const [state, dispatch] = useReducer(toolboxReducer, initialState);
```

Comment:
This feels like a different pattern from most of our patterns, but feels more connected to the interface, which I feel is a good thing.  Should this get documented somewhere as a howto/pattern example?